docker/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM centos:7
+MAINTAINER Apache BookKeeper <dev@bookkeeper.apache.org>
+
+ARG BK_VERSION=4.4.0
+ARG DISTRO_NAME=bookkeeper-server-${BK_VERSION}-bin
+ARG ZK_VERSION=3.5.2-alpha
```

Comment:
It seems the purpose of having ZK is for `metaformat`. except that, I don't see any reason why we need zookeeper in bookkeeper docker image. especially bookkeeper-server package already includes zookeeper dependency -